corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -48,6 +48,7 @@ def add_arguments(self, parser):
         parser.add_argument('--show-diffs', action='store_true', default=False)
         parser.add_argument('--no-input', action='store_true', default=False)
         parser.add_argument('--debug', action='store_true', default=False)
+        parser.add_argument('--run_timestamp', action='store_true', default=None)
```

Comment:
Is it necessary to specify this option to make the migration resumable? If yes, why not enable it by default and (possibly) allow opt-out with an option like `--no-resume`? That suggests the question, is there any reason one would not want to do a resumable migration?

Maybe I'm not understanding the purpose of this argument? A `help="explain what this option does..."` argument would be nice here.

nit: `--run-timestamp` (dash, not underscore) would be more consistent with other options.

Reply:
Nope, I should definitely explain this better:

It'll always be resumable. This option makes it resume to the run started at the specified run_timestamp. 

So if a previous run happened at `1529941054`, that would have been printed at the beginning of the run and the resumable pieces of that run would be keyed on that timestamp, to be picked up whenever.

Reply:
313d238